Source/Scene/Model.js

```diff
@@ -2306,14 +2306,18 @@ define([
                     sampler : sampler,
                     flipY : false
                 });
-            }
-            // GLTF_SPEC: Support TEXTURE_CUBE_MAP.  https://github.com/KhronosGroup/glTF/issues/40
+                // GLTF_SPEC: Support TEXTURE_CUBE_MAP.  https://github.com/KhronosGroup/glTF/issues/40
 
-            if (mipmap) {
-                tx.generateMipmap();
+                if (mipmap) {
```

Comment:
Are you sure about this?  I'm pretty sure it removes the ability to mipmap textures that are defined by buffers, not images.

Reply:
This is the same logic that's in master.  You're saying this be moved out of the if block instead?